en/middleware/micro_dds_edits.md

```diff
@@ -0,0 +1,377 @@
+# MicroDDS
+
+The *PX4-Fast RTPS(DDS) Bridge*, which was also referred to as as the *microRTPS Bridge*, has now been replaced with MicroDDS bridge.
+
+The new approach provides a faster and simpler method of integrating applictaions running and linked in DDS domains (including ROS nodes), making it easy to share sensor data, commands, and other vehicle information.
+
+The following guide describes the new PX4 bridge architecture, and shows the reader how to write a simple *Micro DDS* application to subscribe * publish telemetry updates from the PX4 Autopilot.
+
+:::note
+Micro XRCE-DDS is a software solution that enables communictaion with an existing DDS network. 
+
+:::
+
+## Why this approach is of interest?
+
+Initially, micro-RTPS served as the middleware and its consistent improvement evolved into what's known as the microDDS that provides the user a native interface between the flight controller and the mission computer. 
+The notable difference between the two is what kind of transport protocol each supports. The newest version supports UDP/TCP/Serial/Custom transport protocol.  
+
+The Micro-XRCE-DDS-Gen has 2 major advantages, i.e. it can be used to generate client code and build px4_msgs along with microdds_client together creating a single library.  
+
+:::tip
+
```

Comment:
What's in here?

Reply:
This was initially for my reference and I didn't add it in the SUMMARY thinking it'll be harmless. So it's nothing, thanks for removing it

Reply:
Ah, so I'll look again tomorrow when you've deleted this doc.

The problem with this stuff is that if you commit it, and it does not get deleted it then becomes part of the mainline. The next person to come along then sees two document with similar names - which is the right one, and so on.